Core/GDCore/IDE/Events/EventsRefactorer.cpp

```diff
@@ -555,6 +555,22 @@ void EventsRefactorer::RemoveObjectInEvents(const gd::Platform& platform,
   }
 }
 
+gd::String ReplaceAllOccurrencesCaseInsensitive(gd::String context,
+                                               gd::String from,

```

Comment:
Can you `const gd::String& from` to avoid a (fairly expensive) copy of a string?
Context is "fine" because you modify it.